loops.js

```diff
@@ -33,7 +38,17 @@
 // Tip: een getal is oneven als je het door 2 deelt en er blijft nog iets over. Een getal is even als je het deelt door 2 en er blijft niets over..
 // ==========================================
 
+for (let i = 0; i <= 15; i++) {
+  let unevenEven = i % 2;
 
+  console.log(i, unevenEven ? `is oneven` : `is even`);
+  //   if (unevenEven === 0) {
+  //     console.log(i, "is even");
+  //   }
+  //   if (unevenEven === 1) {
+  //     console.log(i, "is oneven");
+  //   }
+}
```

Comment:
Ik kreeg de code niet gerund. Maar als ik kijk naar je code en ook de code in comments eronder denk ik wel dat je op de goede weg zat. Zelf heb ik hier gebruik gemaakt van de remainder operator (%) in een if else statement om te kijken of er na delen iets overbleef of het antwoord 0 was. In de antwoorden van NOVI is deze operator ook toegepast alleen met een dubbele if statement waardoor je waarschijnlijk minder snel een fout kunt maken. Ik zou hierin het antwoord van NOVI volgen.